controllers/post.controller.js

```diff
@@ -0,0 +1,47 @@
+import Post from '../models/post.model.js';
+import { fMsg } from "../utils/libby.js";
+
+
+
```

Comment:
I think you forgot to retrieve the posts?I recommend to use two functions like getFeed (for whole school which can be seen by everyone) and getAnnoucement (for each class announcement).. but idk just recommendation